test/metabase/integrations/slack_test.clj

```diff
@@ -79,6 +79,12 @@
   (doseq [f [test-no-auth-token test-invalid-auth-token]]
     (f endpoint thunk)))
 
+(deftest slack-app-token-truncation-test
+  (testing "slack-app-token is truncated when fetched by the setting's custom getter"
+    (tu/with-temporary-setting-values [slack-app-token "xoxb-781236542736-2364535789652-GkwFDQoHqzXDVsC6GzqYUypD"]
```

Comment:
this a fake token?

Reply:
yep

Reply:
it's the placeholder one we use on the frontend

Reply:
also just realized tests are failing, fixing them up now